### DISCUSSION TITLE
Can a request driven web service be autoscaled?

### DISCUSSION THREAD

**Discussion by sodhi1981** [aws/copilot-cli#4030] [Q&A]
There is no mention of  cpu_percentage, memory_percentage etc on the manifest page of request driven web services here 
https://aws.github.io/copilot-cli/docs/manifest/rd-web-service/

Is autoscaling available only for load balanced web services?

**Comment by dannyrandall** [comment]
Hey @sodhi1981! Great question. Copilot doesn't currently support configuring autoscaling for Request-Driven Web Services right now 😔. There is an open issue to track this feature request https://github.com/aws/copilot-cli/issues/3200. Give it a 👍!

However, services deployed on App Runner today _do_ get a [default autoscaling configuration](https://docs.aws.amazon.com/apprunner/latest/dg/manage-autoscaling.html) - which I think is 100 requests per instance with a maximum of 25 instances.
